src/plugins/workspace/public/components/workspace_overview/workspace_overview.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useMemo, ReactNode, useEffect } from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiSpacer,
+  EuiFlexItem,
+  EuiText,
+  EuiTabbedContent,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiPanel,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { useObservable } from 'react-use';
+import { i18n } from '@osd/i18n';
+import { CoreStart, PublicAppInfo } from 'opensearch-dashboards/public';
+import { BehaviorSubject } from 'rxjs';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WorkspaceOverviewSettings } from './workspace_overview_settings';
+import { WorkspaceOverviewContent } from './workspace_overview_content';
+import { getStartCards } from './all_get_started_cards';
+import { featureMatchesConfig } from '../../utils';
+import { WorkspaceOverviewCard } from './getting_start_card';
+import { WorkspaceOverviewGettingStartModal } from './getting_start_modal';
+
+export const IS_WORKSPACE_OVERVIEW_COLLAPSED_KEY = 'workspace:overview_collapsed';
+
+export interface WorkspaceOverviewProps {
+  workspaceConfigurableApps$?: BehaviorSubject<PublicAppInfo[]>;
+}
+
+export const WorkspaceOverview = (props: WorkspaceOverviewProps) => {
+  const {
+    services: { workspaces, application, http },
+  } = useOpenSearchDashboards<CoreStart>();
+
+  const currentWorkspace = useObservable(workspaces.currentWorkspace$);
+  const currentWorkspaceId = useObservable(workspaces.currentWorkspaceId$);
+
+  // workspace level setting
+  const workspaceOverviewCollapsedKey = `${IS_WORKSPACE_OVERVIEW_COLLAPSED_KEY}_${
+    currentWorkspaceId || ''
+  }`;
+
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isGettingStartCardsCollapsed, setIsGettingStartCardsCollapsed] = useState(false);
+
+  useEffect(() => {
+    setIsGettingStartCardsCollapsed(localStorage.getItem(workspaceOverviewCollapsedKey) === 'true');
+  }, [workspaceOverviewCollapsedKey]);
```

Comment:
```suggestion
  const [isGettingStartCardsCollapsed, setIsGettingStartCardsCollapsed] = useState(localStorage.getItem(workspaceOverviewCollapsedKey) === 'true');
```
Reduce unnecessary render.